Tests/Data/Elliptic/square_100x100_ComponentTransport/square_1e4_heterogeneity.prj

```diff
@@ -67,9 +41,36 @@
                             </properties>
                         </component>
                     </components>
+                    <properties>
+                        <property>
+                            <name>density</name>
+                            <type>Constant</type>
+                            <value>1e3</value>
+                        </property>
+                        <property>
+                            <name>viscosity</name>
+                            <type>Constant</type>
+                            <value>1e-3</value>
+                        </property>
+                    </properties>
                 </phase>
             </phases>
             <properties>
+                <property>
+                    <name>permeability</name>
+                    <type>Parameter</type>
+                    <parameter_name>kappa1</parameter_name>
+                </property>
+                <property>
+                    <name>porosity</name>
+                    <type>Parameter</type>
+                    <parameter_name>constant_porosity_parameter</parameter_name>
+                </property>
+                <property>
```

Comment:
Is the storage used anywhere? If not you could remove this tags from all project files.

Reply:
Dropped.

Remark: Just retain one in the project file of Theis benchmark. In the project file, it is documented how to consider storage in an indirect way.